src/frontend/admin/src/components/presentational/hook-form/RFHValuesChange.tsx

```diff
@@ -16,15 +18,18 @@ export type RHFValuesChangeProps<T extends FieldValues> = {
 export function RHFValuesChange<T extends FieldValues>({
   debounceTime = 800,
   updateUrl = false,
+  enable = true,
```

Comment:
What do you to explicit it's purpose?

```suggestion
  autoSave = true,
```